### PR TITLE
Reformat linux-networking document for user guide

### PR DESCRIPTION
**Pull request by ffoulkes**
Reformatted the DPDK linux-networking document for inclusion in the user guide.

Intended primarily for demonstration purposes (though it could be merged if deemed appropriate).

This PR was based on https://github.com/ipdk-io/networking-recipe/pull/240 so it could leverage the changes made in the latter PR.

### CODE REVIEW

**Review by nupuruttarwar — APPROVED** [review]
LGTM

File: docs/features/dpdk/linux-networking.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+   to specify the absolute paths of the artifacts (json and spec files).
+
+5. Generate binary image using tdi-pipeline builder:
+
+   ```bash
+   tdi_pipeline_builder --p4c_conf_file=lnw.conf \
+       --bf_pipeline_config_binary_file=lnw.pb.bin
+   ```
+
+## Limitations
```
Comment (ffoulkes) [inline docs/features/dpdk/linux-networking.md:63]: Move Limitations to end of document?

File: docs/features/dpdk/linux-networking.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
+# Linux Networking
+
+<!--
+Copyright 2022-2023 Intel Corporation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+This page describes a step-by-step procedure to run Linux networking scenario.
```
Comment (ffoulkes) [inline docs/features/dpdk/linux-networking.md:8]: This kind of comment is appropriate for a standalone document. It's not appropriate for a user guide.

For the user guide, the initial paragraph(s) should introduce the reader to the **feature**. The page/chapter/document are rarely, if ever, mentioned.

"The Linux Networking feature ..."

File: docs/features/dpdk/linux-networking.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+- Only VNI 0 is supported.
+- We are not supporting any ofproto rules which would not allow for FDB learning on OVS.
+
+## Steps to create the topology
+
+The [gnmi-ctl](/clients/gnmi-ctl) and [p4rt-ctl](/clients/gnmi-ctl) utilities
+used below can be found in $IPDK_RECIPE/install/bin. They should be run with
+`sudo`.
+
+## 1) Bind physical port to IO driver
```
Comment (ffoulkes) [inline docs/features/dpdk/linux-networking.md:83]: Yes, I've backtracked somewhat on the question of numbering paragraphs.

We definitely do not want standard hierarchical numbering. That's appropriate for a reference document, not for a page or chapter in a manual.

But this page has a _loooong_ list of steps, and I'm concerned about the user losing their place in the document.

One of the documents I was read earlier this week -- I think it was part of the ReadTheDocs developer documentation -- said something to the effect that really long pages should generally be broken up into smaller pages. (Rather like really large functions or modules...)

Logistically, you'd either have a much more compact list, each of whose items has a link to a page that documents the item, or you'd have a series of chained pages, each with links to the preceding or following page. (Basically, a doubly-linked list.)

Some of these pages might well be shared -- one on starting OvS, one on starting infrap4d, for example.

If the Networking Recipe variants are all variations of the same document, we might very well want  whole sections that are common.

**Pull request closed without merging**